plugins/kubernetes-backend/src/service/KubernetesProxy.test.ts

```diff
@@ -657,29 +637,135 @@ describe('KubernetesProxy', () => {
         },
       ),
     );
-    permissionApi.authorize.mockResolvedValue([
-      { result: AuthorizeResult.ALLOW },
-    ]);
```

Comment:
Are these permissionApi mocks no longer necessary?

Reply:
Moved up into the `beforeEach` since all tests are allowing.

Reply:
^ very happy to see this by the way!